### PR TITLE
Temporary fix for TFP version

### PR DESCRIPTION
**Pull request by awav**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
     requirements.append("dataclasses")
 
 if not on_readthedocs:
-    requirements.append("tensorflow-probability>=0.9")
+    requirements.append("tensorflow-probability==0.9")
```
Comment (joelberkeley-pio) [inline setup.py:34]: does this allow for e.g. 0.9.1?

Comment (st--) [reply setup.py:34]: No. You can use `~=` for that though.

Comment (joelberkeley-pio) [reply setup.py:34]: that'd be good

Comment (st--) [reply setup.py:34]:
```suggestion
    requirements.append("tensorflow-probability~=0.9")
```

Comment (awav) [reply setup.py]: There is no 0.9.1 and there will not be such thing :)

**Comment by st--** [comment]
Can we please fix it properly instead? I've found the issue - see #1473

**Comment by st--** [comment]
Closing in favour of actually fixing the issue: #1473 (unit tests pass!)

**Pull request closed without merging**